corehq/apps/export/tests/test_export_views.py

```diff
@@ -33,7 +34,7 @@ class FakeDB(object):
 
     @staticmethod
     def get(blob_id):
-        content = 'foo\n'
+        content = b'foo\n'
```

Comment:
Did you find this one manually?  I could see us having hard-to-detect failures when using `BytesIO` with the import of `unicode_literals`.

Reply:
Oh yeah, looks like a test caught it?